scwx-qt/source/scwx/qt/view/level3_product_view.cpp

```diff
@@ -17,7 +17,7 @@
 #include <boost/timer/timer.hpp>
 #include <fmt/format.h>
 
-#if !(defined(_MSC_VER) || defined(__clang__))
```

Comment:
I think this include usage can go away completely. This is no longer used.

It was originally introduced here:
https://github.com/dpaulat/supercell-wx/commit/1e7df9f